SPECS/kernel/kernel-uki.spec

```diff
@@ -13,12 +13,12 @@
 # We need to temporarily turn it off by disabling xsaves until the problem
```

Comment:
Is this comment now outdated? Should we remove it?

Reply:
True. Thank you for the catch